bcrs-ui/src/resources/externalResourceExample.ts

```diff
@@ -0,0 +1,12 @@
+export const ExternalResource = [
+  {
+    id: 1,
+    displayName: 'Cameron',
+    message: 'Hola, Coma Estas?'
```

Comment:
Lol, should be "cómo estás"

Reply:
haha, your very right! This has been fixed 😆

Reply:
De nada